crates/ln-dlc-node/src/lib.rs

```diff
@@ -105,6 +105,57 @@ pub enum PaymentFlow {
     Outbound,
 }
 
+#[derive(PartialEq, Debug, Clone)]
+pub enum ChannelState {
+    Pending,
+    Open,
+    Closed,
+    ForceClosedRemote,
+    ForceClosedLocal,
+}
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Channel {
+    pub id: Option<i32>,
+    // The `user_channel_id` is set by 10101 at the time the `Event::HTLCIntercepted` when
+    // we are attempting to create a JIT channel.
```

Comment:
🔧 All these could be docstrings I think.

Reply:
changed to docstrings.